releasenotes/acs-javascript-calling-library-release-notes.md

```diff
@@ -10,6 +10,24 @@ If you are working with **Teams users**, please follow the `Teams identities` do
 - [Teams Identities quick start](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-with-voice-video-calling-custom-teams-client)
 - [Teams Identities object model](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/voice-video-calling/get-started-with-voice-video-calling-custom-teams-client#azure-communication-services-calling-web-sdk-object-model)
 
+## 1.35.1 (2025-05-08)
```

Comment:
Hello @enricohuang, just a note. Would it be possible to update release notes around the same time the packages are available in npm?